torch/onnx/verify.py

```diff
@@ -0,0 +1,262 @@
+import copy
+import io
+import os
+import tempfile
+from contextlib import ExitStack
```

Comment:
I would avoid importing a class from a module. Import the module instead to keep the name space clean and allow readers more context on where the class comes from (below when being used). https://google.github.io/styleguide/pyguide.html#22-imports